api/views.py

```diff
@@ -79,16 +80,7 @@ def post(self, request, *args, **kwargs):
         return self.create(request, *args, **kwargs)
 
 
-class SalaryListAPIView(mixins.CreateModelMixin, generics.ListAPIView):
-    serializer_class = SalaryListSerializer
-    pagination_class = LimitOffsetPagination
-
-    def get_queryset(self):
-        qs = Salary.objects.filter(company_employee__employee__id=self.kwargs['id'])
-        return qs
-
-    def get(self, request, *args, **kwargs):
-        return self.list(request, *args, **kwargs)
-
-    def post(self, request, *args, **kwargs):
-        return self.create(request, *args, **kwargs)
+class SalaryListAPIView(APIView):
+    def get(self, request, id):
+        add_to_salary_cached.delay(id)
```

Comment:
this should be called when a salary is added for a `user+month` (which is not in `get` I believe)